src/material/schematics/ng-generate/mdc-migration/rules/components/radio/radio-styles.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ClassNameChange, StyleMigrator} from '../../style-migrator';
+
+export class RadioStylesMigrator extends StyleMigrator {
+  component = 'radio';
+
+  deprecatedPrefix = 'mat-radio';
+
+  mixinChanges = [
+    {
+      old: 'radio-theme',
+      new: ['mdc-radio-theme', 'mdc-radio-typography'],
+    },
+  ];
+
+  classChanges: ClassNameChange[] = [
+    {old: '.mat-radio-group', new: '.mat-mdc-radio-group'},
+    {old: '.mat-radio-button', new: '.mat-mdc-radio-button'},
+    {old: '.mat-radio-label-content', new: 'label'},
```

Comment:
This selector seems very unspecific, and could target many other things. Is there a way we can be more specific, scoping the selector to the actual radio label?

Reply:
The label doesn't have anything more specific to select it by https://github.com/angular/components/blob/master/src/material-experimental/mdc-radio/radio.html#L29. Trying to make sure it is scoped within these radio styles might be more work than needed. Internally this is how it is being migrated as well. If this might be an issue, then adding a class to the element is probably the best option.

Reply:
Wouldn't that bear the risk of having something like this:
```
.mat-radio-button {
    .mat-radio-label-content { ... }
}
```
Turn into this:
```
.mat-radio-button {
    .mat-radio-button label { ... }
}
```

I think appending classes in a specific way like this might be a slippery slope with a lot of edge cases, no? Might be better to leave a comment for teams to handle this case, especially considering it's probably fairly rare that someone is styling the radio button label

Reply:
yeah, I just edited my comment after also realizing an issue with :not

I think either not migrating w/ todo, or migrating with a new class for the label would make . The former seems the easiest

Reply:
I think this is another one of those cases where I wish the sass ast was just a little bit more verbose! Being able to parse individual selectors, pseudo classes, etc. would let us handle cases like this so much more gracefully

Reply:
I'm okay if we skip this for external users, since we should end up leaving a TODO for these cases anyways